features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/reflection/ComposeReflection.kt

```diff
@@ -40,6 +40,10 @@ internal object ComposeReflection {
 
     val TextStringSimpleElement = getClassSafe("androidx.compose.foundation.text.modifiers.TextStringSimpleElement")
     val ColorProducerField = TextStringSimpleElement?.getDeclaredFieldSafe("color")
+
+    val BackgroundElementClass = getClassSafe("androidx.compose.foundation.BackgroundElement")
```

Comment:
for all the reflection classes/properties we are using we should add corresponding rules to the `consumer-rules.pro`

Reply:
Good point, `consumer-rules.pro` updated.